Sources/ApollonShared/DataModels/UMLElement.swift

```diff
@@ -117,6 +117,43 @@ public class UMLElement: Codable, SelectableUMLItem {
             self.children?.append(child)
         }
     }
+    //// NEWLY ADDED
```

Comment:
What is this? Is this from cursor?

Reply:
No it was a note to myself, I forgot to delete.

Reply:
Okay let me k is once you have deleted those comments